### DISCUSSION TITLE
Confirm close PR

### DISCUSSION THREAD

**Discussion by alexstewartja** [community/community#7743] [Mobile] [answered]
After swiping up the bottom action sheet on PRs/issues, the `CLOSE` action button becomes an easy target for mistaps. Adding an "Are you sure?" confirmation prompt would solve.

Ref: #3540

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note @alexstewartja — we are currently in the process of moving the `Close` action to the top bar behind the `···` button. This should hopefully reduce mis-taps. Since close actions are non-destructive, we don't plan to add a confirmation prompt.
